### PR TITLE
Component for todo assignment with duedate selection

### PR DESCRIPTION
**Pull request by harish86**

### CODE REVIEW

**Comment by harish86** [comment]
Here is how it looks:

File: web/static/components/cards/to-do-edit.vue [hunk 1]
```diff
@@ -120,14 +114,38 @@
             isDone: task.isDone
           }
         }); //.then( resp => { this.$emit('taskAddOrUpdate', resp.data.data); });
+      },
+      updateAssignment(task, value) {
+        task.userId = value.userId;
+        task.dueDate = value.dueDate;
+        this.updateTask(task);
+        for(var i=0; i<this.companyUsers.length; i++) {
```
Comment (parallel588) [inline web/static/components/cards/to-do-edit.vue]: Hi @harish86, I think better use find method.
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/find

File: web/static/components/shared/inline-datetime-picker.vue [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    components: {
+      'el-date-picker': ElementUi.DatePicker
+    },
+    methods: {
+      onChange(value) {
+        this.$emit('change', value);
+      }
+    },
+    mounted() {
+      console.log(this);
```
Comment (parallel588) [inline web/static/components/shared/inline-datetime-picker.vue]: need to remove

File: web/static/components/shared/inline-datetime-picker.vue [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      this.$refs.dateTimePicker.showPicker();
+      this.$refs.dateTimePicker.handleClick = function(item) {
+        alert(1, item);
```
Comment (parallel588) [inline web/static/components/shared/inline-datetime-picker.vue]: ?

**Pull request merged**